.gitignore

```diff
@@ -105,3 +105,6 @@ wp-tests-config.php
 
 # Visual regression test diffs
 tests/visual-regression/specs/__snapshots__
+
```

Comment:
This can be just added in the `# Files and folders related to build/test tools` section near the top without a new comment.

Reply:
moved